docs/release-notes/api-cli-access-token.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+**New Features**
+
+-  API/CLI: Add the ability to create and administer access tokens for users in automated
+      workflows. Users can define the lifespan of these tokens, making it easier to securely
+      authenticate and run processes. This feature enhances automation while maintaining strong
+      security protocols by allowing tighter control over token usage and expiration.
```

Comment:
suggested edits sent via slack

Reply:
I have updated the docs with the later commit. Let me know